core/senders/qq_dispatcher.py

```
@@ -234,28 +274,76 @@ async def dispatch_processed_batches_to_targets(
+                            else:
+                                await send_processed_batch_fn(
+                                    batch_data=chunk_send_batches[0],
+                                    unified_msg_origin=unified_msg_origin,
+                                    self_id=self_id,
+                                    node_name=node_name,
+                                    target_session=target_session,
+                                    allow_forward_nodes=False,
+                                )
+                            big_merge_outcome = "success"
+                            break
+                        except Exception as e:
```

Ruff (0.16.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)